content/docs/use-cases/ci-cd-for-machine-learning.md

```diff
@@ -0,0 +1,98 @@
+# Continuous Integration and Deployment for Machine Learning
+
+Applying DevOps methodologies to machine learning (MLOps) and data management
+(DataOps) is increasingly common. This means resource orchestration
+(provisioning servers for model training), model testing (validating model
+inference), and model deployment to production, as well as monitoring &
+feedback. [DVC](/) can manage data/models and reproducible pipelines, while
+[CML] can assist with orchestration, testing and monitoring.
+
+[cml]: https://cml.dev
+
+![](https://static.iterative.ai/img/ml-pipeline.png) _Basic ML pipeline_
+
+The main benefits of CI/CD (continuous integration and continuous delivery) for
+ML are detailed below.
+
+Firstly, we can automate and enforce testing. This includes enforcing integrity
```

Comment:
I'm a bit confused about the story. Are we focusing CI/CD for ML per se or how DVC and CML instrument it? I'm guessing the latter (though the former could be a great blog topic). In the intro I think that it's implied that the reader already is convinced of the benefits of CML for them. And if I'm right, we're probably going into too much detail about benefits of CI/CD for ML (3 full bullet lists).

That said, a good paragraph summarizing all of them could definitely help cement the intro and drive some search results.